challenge.md

```diff
@@ -6,39 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git clone 'url'
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git diff
 
 3. Oops - it looks like there was a change that we forgot to include in our last commit. How can we revise that commit and fix things?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git checkout x
 
 4. It looks like there was a change on the original repo that we forked from. How could we grab those changes and incorporate them into our local repo?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+$ git reset --hard commit-name
```

Comment:
Oh no - please don't do that! `git reset --hard` will roll back Git _and your code files_ to where they were at some point in the past. 

What we want to do here is retrieve changes from an upstream repo. Assuming we've already set an upstream (using `git remote add upstream some_url`), the command to retrieve the latest version of our upstream is `git pull upstream` (plus, optionally, the specific branch that we'd like to pull from).